core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/store/dot-experiments-configuration-store.ts

```diff
@@ -273,6 +274,32 @@ export class DotExperimentsConfigurationStore extends ComponentStore<DotExperime
         }
     );
 
+    readonly deleteGoal = this.effect((goalLevel$: Observable<GoalsLevels>) => {
+        return goalLevel$.pipe(
+            withLatestFrom(this.state$),
```

Comment:
Again... send the experiment info from the component.

Reply:
Fixed.